components/CardList.tsx

```diff
@@ -0,0 +1,69 @@
+import CardItem from "./CardItem";
+import { styled } from "styled-components";
+import { Link as TLink } from "@/lib/Type";
+import Link from "next/link";
+
+interface CardListProps {
+  links?: TLink[];
+}
+
+const CardList = ({ links = [] }: CardListProps) => {
+  return (
+    <Container>
+      <ListContent>
+        <ListInner>
+          {links.length > 0 ? (
+            links.map((link) => {
```

Comment:
```suggestion
            links.map(({ id, url} ) => {
```
이렇게 destruction해주면 `link.id`, `link.url` 등으로 써주지않아도되겠죠?